code/ui/components/package.json

```diff
@@ -71,6 +71,7 @@
   },
   "devDependencies": {
     "@popperjs/core": "^2.6.0",
+    "@prettier/sync": "^0.3.0",
```

Comment:
Why do we need this?

Reply:
I don't think we need this. Potentially a leftover when Norbert was experimenting.